install/kubernetes/helm/istio/charts/grafana/templates/gateway.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.ingressgateway.enabled -}}
+{{- $serviceName := include "grafana.fullname" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: grafana-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "{{- .Values.ingressgateway.host }}"
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: grafana
+spec:
+  host: grafana.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
The `cluster domain` is also configurable, shall we introduce a new parameter in `values.yaml` for `cluster.local`?

Reply:
maybe. i think all of this will end up being refactored when we realign the gateway strategy, so i think this is OK to leave as-is for the time being.